src/audits/manifest/short-name-length.js

```diff
@@ -50,14 +50,17 @@ class ManifestShortNameLength extends Audit {
     let debugString;
     const manifest = artifacts.manifest.value;
     const suggestedLength = 12;
+    const shortNameValue = manifest && manifest.short_name && manifest.short_name.value;
```

Comment:
yep :) I still like just a function or something but a ternary is nicer than an && chain